test/backend/test_node.py

```diff
@@ -113,6 +114,22 @@ def test_arg_max(self):
         result = data.shape[axis] - result - 1
         np.testing.assert_almost_equal(output["reduced"], result)
 
+      # test tensor(bfloat16) for opset 13
+      if not legacy_opset_pre_ver(13):
+        node_def = helper.make_node("ArgMax", ["data"], ["reduced"],
+                                    axis=axis,
+                                    keepdims=0)
+        
+        np_fp32 = self._get_rnd_float32(shape=[10, 10])
+        little_endisan = sys.byteorder == 'little'
+        np_uint16_view = np_fp32.view(dtype=np.uint16)
+        np_bfp16 = np_uint16_view[1::2] if little_endisan else np_uint16_view[0::2]
+        data = np_bfp16
```

Comment:
The data seems in np.uint16. Can you please elaborate how this is going to test and verify bfloat16 handling in the handler?

Reply:
@chinhuang007 bfloat16 is very similar to float32 except that it has less precision. So the approach which I leveraged from Cast here: https://github.com/onnx/onnx/blob/master/onnx/backend/test/case/node/cast.py (see lines 41 to 46) basically starts with a float32 and construct a view of the memory but as uint16. Once we do that, we need take care of the byte order. Basically manually creating a bfloat16. Hope that helps. Thanks

Reply:
@chinhuang007 thanks on the discussion on support for bfloat16. I am reworking this. Thanks again.